tuxemon/resources/db/encounter/.~lock.JSON-o-matic for Encounters.ods#

```diff
@@ -0,0 +1 @@
+,BILLACER/Bill,BillAcer,08.08.2017 22:25,file:///C:/Users/Bill/AppData/Roaming/LibreOffice/4;
```

Comment:
This file, `tuxemon/resources/db/encounter/.~lock.JSON-o-matic for Encounters.ods# `, should not be included in the commit.